sdk/devcenter/Azure.Developer.DevCenter/CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
-### Features Added
+## 1.0.0 (Unreleased)
+This release updates the Azure DevCenter library to use the 2023-04-01 GA API.
 
 ### Breaking Changes
 
-### Bugs Fixed
+ - `EnvironmentsClient` renamed to `DeploymentEnvironmentsClient`
+ - `DevBoxesClient` and `DeploymentEnvironmentsClient` no longer accepts project as a constructor parameter
+ - `DeploymentEnvironmentsClient` now works with "environment definitions" instead of "catalog items"
+ - Createing a new environment requires passing `environmentDefinitionName` instead of `catalogItemName`
+ - Createing a new environment requires passing an additional parameter `catalogName`
+
+### Features Added
 
-### Other Changes
+ - Tests and samples demonstrating the new API
```

Comment:
Nit: Tests and samples aren't considered changes in the public API, so there's no need to include them in the changelog.

Reply:
Removed